### PR TITLE
174 Release Notes

### PR DESCRIPTION
**Pull request by michaelranaldo**

### CODE REVIEW

File: release-notes/release-notes-174.adoc [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+== Security Fixes
+
+* https://github.com/payara/Payara/pull/2097[PAYARA-2109 - Protect Admin
+Console Web Pages against Clickjacking/UI Redress attacks]
+* https://github.com/payara/Payara/pull/2023[PAYARA-2151 - Backport fix
+for CVE-2017-12615 in Tomcat Code Base]
+* PAYARA-2152 - Confirm Payara is unaffected by CVE-2017-12616
+
+[[upstream-bugs]]
+== Upstream Bugs
```
Comment (OndrejM) [inline release-notes/release-notes-174.adoc]: Please change the title to **Upstream Bug Fixes**. I also changed the script in the wiki to reflect that.

File: release-notes/release-notes-174.adoc [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+* https://github.com/payara/Payara/pull/2024[PAYARA-2161 - Upgrade Weld
+to 2.4.5.Final]
+* https://github.com/payara/Payara/pull/2099[PAYARA-2234 - Upgrade JUnit
+to 4.12]
+
+[[known-issues]]
+= Known Issues
+
+Known issues can be seen on our GitHub issues page here:
+https://github.com/payara/Payara/issues(https://github.com/payara/Payara/issues).
```
Comment (OndrejM) [inline release-notes/release-notes-174.adoc]: With asciidoc, you can remove the link in brackets. It's not necessary and the current format doesn't work.

File: security/security-fix-list.adoc [hunk 3]
```diff
@@ -8,10 +8,12 @@ can or have impacted Payara Server across releases:
 [cols="<,<,<,<,<,<",options="header",]
 |=======================================================================
 |ID |Status |Summary |Release |Pull Requests |Observations
+|https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-12616[CVE-2017-12616] | N/A | Apache Tomcat security constraint bypass and VirtualDirContext vulnerability | 4.1.2.174 | N/A | Unrelated to Payara Server
```
Comment (OndrejM) [inline security/security-fix-list.adoc]: PAYARA-2109 is missing - it's mentioned in the release notes, but doesn't have CVE ID. As an ID we can put [Clickjacking_Defense_Cheat_Sheet](https://www.owasp.org/index.php/Clickjacking_Defense_Cheat_Sheet) and as a description the title of the PAYARA-2109 issue.

Comment (michaelranaldo) [reply security/security-fix-list.adoc]: Yes, I didn't know to include it if it didn't have a formal CVE. I'll add it now

**Pull request merged**